specification/documentation/APIDescription.md

```diff
@@ -196,4 +196,20 @@ The channels can have the following supplier statuses:
 * `notified` - a push notification is reported as having been successfully relayed to one or more devices
 * `received` - the request has been received by the supplier and is queued to be processed
 
-Statuses for other suppliers will be added in future.
+### GOV.UK Notify (Email/SMS)
+
+* `delivered` - the message has been successfully delivered to the user
+* `permanent_failure` - the Email/SMS provider could not deliver the message, this can happen if the phone number was wrong or if the network operator rejects the message
+* `temporary_failure` - the Email/SMS provider could not deliver the message, this can happen when the recipient’s phone is off, has no signal, or their text message inbox is full
+* `technical_failure` - the message was not sent because there was a problem between GOV.UK Notify and the Email/SMS provider
+
+### GOV.UK Notify (Letters)
```

Comment:
I think the supplierStatus enum that feeds the get message status schema will also need updating 
![image](https://github.com/user-attachments/assets/dde819f6-90a4-4f9a-8df4-75e6d76d5fd1)

Reply:
Done.